comms/storage/storageserver/static/status.html

```diff
@@ -1,18 +1,20 @@
 <!doctype html>
```

Comment:
(no real changes here) this somehow got formatted, which I didn't mean to do. not sure if this is a style you prefer, so I can revert if you want?